storyblok-vercel-configuration.md

```diff
@@ -0,0 +1,86 @@
+# Production, Staging, Preview configuration
+
+Storyblok = place where content is stored (Headless CMS = content management system)
+
+GitHub (this repo) = place where code is stored (hosted SVC = software version control)
+
+Vercel/Next.js = service that generates and deploys website by combining content + code.
+
+The following document uses Quansight Consulting "LLC" as an example, but it also applies to
+Labs: wherever you come across the string `consulting` below, you just have to
+replace it with `labs`.
+
+## Production = Live Public-Facing Site
+
+Currently deploys to: quansight-consulting.vercel.app
+
+Later, will deploy to: quansight.com
+
+Triggered by:
+
+- adding new commits (e.g. merging a PR) to `main` branch of [Quansight-website
+  repo](https://github.com/Quansight/Quansight-website)
+- Promote to Production button in vercel.com admin
```

Comment:
Where is this button? I can't find it in the Vercel admin. Maybe I don't have permission to see it?

Reply:
![image](https://user-images.githubusercontent.com/68534835/174504438-35acbd1e-12b0-43af-bb1d-4d3d315bf367.png)

It's here. But we should not use this option. Releasing changes **in code** should be done by merging code from `develop` branch to `main` with version bump in project.json file.